### ISSUE TITLE
Reopen the paczkomat after initial opening

### ISSUE THREAD

**Issue by jpruciak** [loboda4450/inpost-telegram-bot#3]
like above

**Comment by loboda4450** [comment]
It is on my list, but cannot do a single thing without any parcel waiting in paczkomat :/

**Comment by jpruciak** [comment]
I have one paczka in paczkomat if you want! You could sniff it and open it when I'm near the paczkomat.

**Comment by loboda4450** [comment]
Thanks for your help @jpruciak with sniffing. It should be ready till end of the week!

**Comment by jpruciak** [comment]
Yes, paczka has been wyjęta from the paczkomat and packets got złapaned. Paczkomat felt like a broken szafka when opening so many times.

**Comment by loboda4450** [comment]
btw @jpruciak it is partially wrong repo I guess, it should be posted [@IFOSSA/inpost-python](https://github.com/IFOSSA/inpost-python), cuz it is not included in library yet 😆

**Comment by jpruciak** [comment]
no fucks given bout lib, I need it on bot so I post issue on the bot, If I need it in the lib for other purposes I'd open it on the lib repo
